src/visning/Visning.less

```diff
@@ -0,0 +1,15 @@
+.Visning {
```

Comment:
Hvilken konvensjon har vi for classnames i CSS?
Stor eller liten forbokstav?

Reply:
Vi har brukt liten forbokstav før, altså camelCase

Reply:
FIkser